### ISSUE TITLE
[Planting Flow] Redesign New Tree Sidebar to follow same design principles to the tree detail sidebar

### ISSUE THREAD

**Issue by ri0nardo** [waterthetrees/wtt_front#467]
Since the tree detail sidebar is being scoped out. This is a good chance to have the sidebars all follow the same design system, so its consistent overall. There isn't much to change except there should be an option to take a picture of the tree in the future. and add what type of planting occurred during that session. So if a tree that was planted was larger and needed stakes, that could be noted.

Resources:
https://dribbble.com/shots/17813322--Navigation-GPS-app-Walk-Run-Bike/attachments/12984481?mode=media

![Image](https://user-images.githubusercontent.com/98299952/218299131-bbeb7b2d-f90e-4ddc-8e9d-0e6d8b102e9a.PNG)

![image2](https://user-images.githubusercontent.com/98299952/226790633-1fd889a5-d77c-4af8-af3a-6254ab9de060.png)
![image1](https://user-images.githubusercontent.com/98299952/226790638-1efecf7c-b2a8-48bc-bf50-cf25f0ef4399.png)
![image0](https://user-images.githubusercontent.com/98299952/226790639-970a2e39-66c2-4d14-8000-caebb3727c68.png)

**Comment by Dawoodeveloper** [comment]
Can i solve this issue?
Please assign this issue to me. so i will contribute in this issue.

**Comment by ri0nardo** [comment]
@zoobot question about the "planter" section. Is the intent for the future to have multiple people added when planting a tree? Say there is a child and adult planting trees together, they both can get credit? For a new section, do we want to have an organization tag? So if the planter is from "FUF" it would appear?

**Comment by zoobot** [comment]
Multiple people field is a great idea as street trees are usually planted by groups of people.
We do want the organization field to stay. Not sure how multiple people would work with the login. Maybe we could have invites or tag friends or something. We basically going to have to build a social network... :)

**Comment by ri0nardo** [comment]
> Multiple people field is a great idea as street trees are usually planted by groups of people. We do want the organization field to stay. Not sure how multiple people would work with the login. Maybe we could have invites or tag friends or something. We basically going to have to build a social network... :)

How im thinking of the feature is similar to how github can assign people to a ticket. So you can tag a person to a tree that participated in the planting / removal or a tree. But I understand that is not really a feature that needs to be added now. Its a nice to have.

**Comment by ri0nardo** [comment]
Redesigned the add tree sidebar to follow a similar design to the tree info sidebar. @zoobot what do you think? I have to modify a few things to make it compatible with accessibility, and make the submit section more visible. 
![Desktop View](https://user-images.githubusercontent.com/98299952/227107125-1022f96f-5c5f-431d-aac0-715ea829cef9.jpg)
![Frame 162](https://user-images.githubusercontent.com/98299952/227107147-7ce73f0f-b661-47c8-b470-c0b61859a31c.jpg)

**Comment by ri0nardo** [comment]
@zoobot I changed the input fields so they now all fit on a desktop screen without scrolling currently. I did change the order of the information section to have the tree health / status as the first field. My thought process was that if the information that the person is providing isn't a tree (if its a vacant spot, or dead tree), they wouldn't have a reason to input a tree species etc.
Is the order of the input fields ok or do we still want the tree naming to come first? Also do we need to have the "volunteer email" if the person who is inputting the information is already confirmed by signing in?
![Desktop View](https://user-images.githubusercontent.com/98299952/227412434-6de615a4-8338-4718-80e5-7219b03ca1f4.jpg)
![Frame 164](https://user-images.githubusercontent.com/98299952/227412443-dab5b89b-3c5a-4489-bc9a-51a18c2e1a6a.jpg)
